common/src/main/java/com/alibaba/nacos/common/utils/MD5.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.alibaba.nacos.client.config.utils;
```

Comment:
The utility class is already at the end of Utils

Reply:
sorry,I don't understand what you mean. the utility class is move from client module to common module. do you mean  name  should like MD5Utils  end with Utils?

Reply:
yes